cmd/cli/install_test.go

```diff
@@ -527,6 +528,29 @@ var _ = Describe("Test envoy log level types", func() {
 	})
 })
 
+var _ = Describe("Test Envoy proxy sidecar image regex", func() {
+	Context("Test isValidEnvoyImage", func() {
+		It("Should validate if the specified Envoy image is of the right form", func() {
+			err := isValidEnvoyImage("envoyproxy/envoy-alpine:v1.17.2")
+			Expect(err).NotTo(HaveOccurred())
+
+			err = isValidEnvoyImage("envoyproxy/envoy-alpine:v1.17.1")
+			Expect(err).NotTo(HaveOccurred())
+
+			err = isValidEnvoyImage("envoyproxy/envoy-alpine:v1.17.2")
+			Expect(err).NotTo(HaveOccurred())
+		})
+
+		It("Should correctly error for invalid envoy log level", func() {
```

Comment:
```suggestion
		It("Should correctly error for invalid form of envoy image ", func() {
```

Reply:
Removed based on the comment from @shashankram to remove the `envoy-image` option from the install CLI